Problem with CodeAnalysisRules when using ";" (semicolon)

I've been trying to run some SQL Code Analysis at 2.7.0 version and I've found a problem with defining CodeAnalysisRules using ";" (semicolon). Apparently it fails to build on linux\macos machine, but it works on windows. I assume that it is related to the fact that ";" (semicolon) is a command separator in bash. Here is a part of output:

`Analyzing package '/workspaces/.../Example/obj/Debug/netstandard2.0/Example.dacpac'
Successfully analyzed package 'obj/Debug/netstandard2.0/Example.dacpac'
/usr/bin/sh: 2: /tmp/MSBuildTempvscode/tmp63c87377fa93446488be8bc7aa3855d4.exec.cmd: -SqlServer.Rules.SRN0007: not found
/home/vscode/.nuget/packages/msbuild.sdk.sqlproj/2.7.0/Sdk/Sdk.targets(244,5): error MSB3073: The command "dotnet "/home/vscode/.nuget/packages/msbuild.sdk.sqlproj/2.7.0/Sdk/../tools/net6.0/DacpacTool.dll" build -o "obj/Debug/netstandard2.0/Example.dacpac" -n "Example" -v "1.0.0" -sv SqlAzure -i "obj/Debug/netstandard2.0/Example.InputFiles.txt" -r "/home/vscode/.nuget/packages/microsoft.sqlserver.dacpacs.azure.master/160.0.0/tools/master.dacpac;dbl=master|dbv=|srv=;"   -dp IncludeCompositeObjects=true          -an -ar -SqlServer.Rules.SRP0005;-SqlServer.Rules.SRN0007" exited with code 127.`

See the part where "**/usr/bin/sh**" is trying to find "-SqlServer.Rules.SRN0007" command... 

The problem is only when using ";" (semicolon) in CodeAnalysisRules:
`<CodeAnalysisRules>-SqlServer.Rules.SRP0005;-SqlServer.Rules.SRN0007</CodeAnalysisRules>
`

I've figured it out, that there is a dirty fix to that problem (and it works even on windows).
**Just add extra "" (quotation)**: 
`<CodeAnalysisRules>"-SqlServer.Rules.SRP0005;-SqlServer.Rules.SRN0007"</CodeAnalysisRules>
`

--- a/src/DacpacTool/DacpacTool.csproj
+++ b/src/DacpacTool/DacpacTool.csproj
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
@@ -13,8 +13,8 @@
 
   <ItemGroup>
     <!-- These packages contain DacFX analysis rules, which must be located with the executable in order to be discovered-->
-    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.0-preview4" IncludeAssets="runtime" />
-    <PackageReference Include="ErikEJ.DacFX.TSQLSmellSCA" Version="1.0.0-preview1" IncludeAssets="runtime" />
+    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.0-preview7" IncludeAssets="runtime" />
+    <PackageReference Include="ErikEJ.DacFX.TSQLSmellSCA" Version="1.0.0-preview2" IncludeAssets="runtime" />
       
     <PackageReference Include="Microsoft.SqlServer.DacFx" Version="162.2.111" />
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />

--- a/src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets
+++ b/src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets
@@ -230,7 +230,7 @@
       <PostDeploymentScriptArgument>@(PostDeploy->'--postdeploy %(Identity)', ' ')</PostDeploymentScriptArgument>
       <RefactorLogScriptArgument>@(RefactorLog->'--refactorlog %(Identity)', ' ')</RefactorLogScriptArgument>
       <RunSqlCodeAnalysisArgument Condition="'$(RunSqlCodeAnalysis)' == 'True'">-an</RunSqlCodeAnalysisArgument>
-      <CodeAnalysisRulesArgument Condition="'$(CodeAnalysisRules)'!=''">-ar $(CodeAnalysisRules)</CodeAnalysisRulesArgument>
+      <CodeAnalysisRulesArgument Condition="'$(CodeAnalysisRules)'!=''">-ar &quot;$(CodeAnalysisRules)&quot;</CodeAnalysisRulesArgument>
       <DebugArgument Condition="'$(MSBuildSdkSqlProjDebug)' == 'True'">--debug</DebugArgument>
       <TreatTSqlWarningsAsErrorsArgument Condition="'$(TreatTSqlWarningsAsErrors)' == 'True' Or ('$(TreatWarningsAsErrors)' == 'True' And '$(TreatTSqlWarningsAsErrors)' == '')">--warnaserror</TreatTSqlWarningsAsErrorsArgument>
       <GenerateCreateScriptArgument Condition="'$(GenerateCreateScript)' == 'True'">--generatecreatescript</GenerateCreateScriptArgument>

